src/userActions.js

```diff
@@ -0,0 +1,26 @@
+export default class UserActions {
+  // Mark Task As Completed & Update Its Object's Value For Completed Key.
+  completeTask = (items) => {
```

Comment:
Upon reloading the page, the tasks which are marked complete don't show such for their checkboxes. Kindly fix this issue. I recorded [this video](https://www.loom.com/share/585bffd6db9d4f3e8c575f7326d3ea88) where I demonstrated the issue.

Reply:
I have made the changes. Thanks for your review.